tox.ini

```diff
@@ -0,0 +1,24 @@
+[flake8]
+exclude =
+    # No need to traverse our git directory
+    .git,
+    # There's no value in checking cache directories
+    __pycache__,
+    # This contains our built documentation
+    build/*,
+    # This contains builds of flake8 that we don't want to check
+    dist/*
+max-line-length = 120
+show-source = true
+statistics = true
+
+docstring-quotes = """
+multiline-quotes = """
+
+[isort]
+force_single_line=true
+force_sort_within_sections=true
+line_length=119
+
+[pytest]
+addopts = --durations=10 -v --cache-clear
```

Comment:
Should we be running flake8 and pylint here?

Reply:
We could, if that's preferred. I'll make this change.

Reply:
Also, the only linter here is flake8. pylint seems like it will be separate work.